recipes/tn93/meta.yaml

```diff
@@ -0,0 +1,18 @@
+package:
+  name: tn93
+  version: 1.0.2
+
+source:
+  git_rev: v1.0.2
+  git_url: https://github.com/veg/tn93.git
+
+requirements:
+  build:
+    - gcc
```

Comment:
@ericenns can you adopt this please to our guidelines: https://github.com/bioconda/bioconda-recipes/blob/master/GUIDELINES.md#cc
Thanks!